controllers/servers/utils.py

```diff
@@ -846,3 +847,7 @@ def get_replication_object_type_and_id_info(request):
             raise ValidationException(messages.UNSUPPORTED_REPLICATION_SOURCE_TYPE_MESSAGE)
     object_id_info = get_object_id_info(object_id, object_type)
     return object_type, object_id_info
+
+
+def is_call_home_enabled():
+    return getenv(servers_settings.ENABLE_CALL_HOME_ENV_VAR, 'true') == 'true'
```

Comment:
Does this mean that call home will be disabled by default? I couldn't find where this ENV is set

Reply:
it's enabled by default, it comes from the [operator](https://github.com/IBM/ibm-block-csi-operator/pull/342)

Reply:
OK, thanks, [commented](https://github.com/IBM/ibm-block-csi-operator/pull/342#issuecomment-1592887227) there